app/enterprise/0.35-x/kong-manager/networking/configuration.md

```diff
@@ -52,7 +52,49 @@ To enable authentication, configure the following properties:
 on to enforce authorization rules. Otherwise, whoever can log in 
 to Kong Manager can perform any operation available on the Admin API.
 
+## TLS Certificates
+
+By default, if Kong Manager’s URL is accessed over HTTPS _without_ a certificate issued by a CA, it will 
+receive  a self-signed certificate that modern web browsers will not trust, preventing the application 
+from accessing the Admin API.
+
+In order to serve Kong Manager over HTTPS,  use a trusted certificate authority to issue TLS certificates, 
+and have the resulting `.crt` and `.key` files ready for the next step.
+
+1) Move `.crt` and `.key` files into the desired directory of the Kong node.
+
+2) Point [`admin_gui_ssl_cert`] and [`admin_gui_ssl_cert_key`] at the absolute paths of the certificate and key.
+
+```
+admin_gui_ssl_cert = /path/to/test.crt
+admin_gui_ssl_cert_key = /path/to/test.key
+```
+
+3) Ensure that `admin_gui_url` is prefixed with `https` to use TLS, e.g.,
+
+```
+admin_gui_url = https://test.com:8445
+```
+
+### Using `https://localhost`
+
+If using `localhost` for testing, it may be preferrable to just use HTTP as the protocol and, if using RBAC, 
+to set `cookie_secure=false` in Session configuration. The reason to use HTTP for `localhost` is that 
```

Comment:
This feels like a run-on sentence, even though technically it is not. Suggest,
"If serving Kong Manager on `localhost`, it may be preferable to use HTTP as the protocol. If also using RBAC, set `cookie_secure=false` in `admin_gui_session_conf`." (Note: this suggestion also fixes the spelling error in "preferable.")